apps/api/src/migration/1655907719963-AddNewColumnOrganizationToEmployeeTable.ts

```diff
@@ -0,0 +1,41 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddNewColumnOrganizationToEmployeeTable1655907719963 implements MigrationInterface {
+  name = 'AddNewColumnOrganizationToEmployeeTable1655907719963';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Why do we have to recreate id column?

Reply:
We do not need to, But here employee table creates using OOps And was generated using global Typeform and not API/packages Typeform version. So it won't match due to a different version one for OOPS and one for row query.

What cory has generated is the same way for all the other table and contain the same constraints id throughout all the env. Once we merge this migration, then now onwards it will identify employee table changes(new ones only) correctly.